src/main/java/uk/ac/cam/cl/dtg/segue/api/AuthenticationFacade.java

```diff
@@ -433,6 +446,37 @@ public final Response userLogout(@Context final HttpServletRequest request,
         }
     }
 
+    /**
+     * End point that allows the user to logout of other sessions.
+     *
+     * @param request
+     *            so that we can replace the associated session
+     * @param response
+     *            to tell the browser to update the session for segue.
+     * @return successful response to indicate other sessions were invalidated.
+     */
+    @POST
+    @Produces(MediaType.APPLICATION_JSON)
+    @Consumes(MediaType.WILDCARD)
+    @Path("/logout_elsewhere")
+    @ApiOperation(value = "Invalidate other sessions for the current user and create a new session.")
+    public final Response userLogoutElsewhere(@Context final HttpServletRequest request,
+                                     @Context final HttpServletResponse response) {
+        try {
+            this.getLogManager().logEvent(this.userManager.getCurrentUser(request), request, SegueLogType.LOG_OUT_ELSEWHERE,
+                    Maps.newHashMap());
+            SegueMetrics.LOG_OUT_ELSEWHERE.inc();
+
+            userManager.logoutElsewhere(request, response, true);
```

Comment:
When we call logout elsewhere we invalidate the current session. The log gets the current user from the request and validates it. If called after it will fall back on using a non registered user. I've now changed it to extract the user first, call logout elsewhere and then add to the log.